README.md

```diff
@@ -4,33 +4,64 @@ Software Engineering Day1 Assignment
 #Part 1: Introduction to Software Engineering
 
 Explain what software engineering is and discuss its importance in the technology industry.
-
+Software engineering is the application of engineering principles to design,develop,test and deploy softwate systems.
```

Comment:
Software engineering is the branch of computer science that deals with the design, development, testing, and maintenance of software applications. Software Engineering is the application of engineering concepts, techniques and methods to the development of mission-critical software systems.
The software engineering major builds on the computer science major with advanced course work in software architecture and design, software metrics, verification and validation, requirements analysis and specification and the software engineering process.
A person who has successfully gained knowledge on software engineering is called A SOFTWARE ENGINEER.
Successful engineers know how to use the right programming languages, platforms, and architectures to develop everything from computer games to network control systems. In addition to building their own systems, software engineers also test, improve, and maintain software built by other engineers.

Reply:
The field of software engineering offers numerous benefits to both individuals and businesses. For individuals, it provides excellent career prospects, competitive salaries, and opportunities for growth and career advancement. Software engineers are in high demand, and their skills are highly transferable across various industries.

Furthermore, software engineering plays a critical role in the success of businesses in the tech industry. Well-designed and well-developed software solutions can enhance productivity, streamline processes, and improve customer satisfaction. By investing in software engineering, businesses can gain a competitive edge in the market and achieve their long-term goals.